### PR TITLE
Problem 12658

### PR DESCRIPTION
**Pull request by Gor397**

### CODE REVIEW

File: scenes/enumerated/problem_12658/text.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from constants import LanguageConfig
+
+if LanguageConfig.is_armenian():
+    from .armenian_text import *
+else:
+    from .english_text import *
+
+# TODO add __all__ variable
```
Comment (zidder) [inline scenes/enumerated/problem_12658/text.py:8]: Please, do

File: scenes/enumerated/problem_12658/problem_12658.py [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+        for obj in self.mobjects:
+            if obj != taskNumber:
+                temp.append(obj)
+        self.play(*[FadeOut(obj) for obj in temp])
+        self.wait()
+
+        task5 = Tex("$4$", "$a$", "$^2$", "$b$", "$^5$", " $+$ ", "$6$", "$a$", "$^3$", "$b$", "$^5$", font_size=110)
+        self.play(Write(task5))
+
+        self.wait(2)
```
Comment (zidder) [inline scenes/enumerated/problem_12658/problem_12658.py:514]: Can you fix this absent newline?

File: scenes/enumerated/problem_12658/problem_12658.py [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+        self.remove(temp)
+        self.play(Write(task3_copy[-1].set_color(WHITE)))
+        self.wait()
+
+        self.play(
+            ModifyFormula(
+                task3,
+                add_after_items=[len(task3)-1],
+                add_items_strs=[[" $=$ ", "$3$", "$x$", "$y$", "$^3$", "$($", "$2$", "$y$", "$^2$", " $-$ ", "$3$", "$x$", "$^2$", "$)$"]],
+                add_items_colors=[[BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK, BLACK]],
```
Comment (zidder) [inline scenes/enumerated/problem_12658/problem_12658.py:412]: You can store the first list in a variable, then do `[BLACK] * len(variable)`

File: scenes/enumerated/problem_12658/problem_12658.py [hunk 4]
```diff
@@ -0,0 +1,514 @@
...
+                2, 3, 4, 5, 6, 7, 8, 9, 10,
+                21, 22, 23, 24, 25, 26
+            ]
+        )
+        self.wait()
+
+        self.play(
+            ModifyFormula(
+                task2_copy,
+                replace_items=[[0, 1, 2, 3, 4, 5, 6, 7, 8], [12, 13, 14, 15, 16, 17, 18, 19, 20], [22, 23, 24, 25, 26]],
```
Comment (zidder) [inline scenes/enumerated/problem_12658/problem_12658.py:273]: Can you add comments after those modifies for your future self?

**Review by zidder — DISMISSED** [review]

**Review by narek-mkrtumyan — APPROVED** [review]

**Pull request merged**